config/default.yml

```diff
@@ -489,6 +489,9 @@ Layout/SpaceBeforeBlockBraces:
   SupportedStyles:
     - space
     - no_space
+  SupportedStylesForEmptyBraces:
```

Comment:
That was made intentionally to avoid breaking changes. I'd prefer to fallback to `EnforcedStyle` if `EnforcedStyleForEmptyBraces` is missing.